src/content/pages/build/taco.md

```diff
@@ -0,0 +1,213 @@
+---
+template: build-page/taco
+path: /build/taco
+title: TACo Page
+seoTitle: Threshold Taco
+seoDescription: "TACo (Threshold Access Control) enables end-to-end encrypted data sharing and communication."
+tacoInfo:
+  title: "TACo: Privacy without trust"
+  description: "Give your users true sovereignty over their data, today. TACo – the first and only access control layer you aren’t forced to trust."
+  image: /images/taco-encryption.svg
+  buttons:
+    - label: Start Building
+      url: https://docs.threshold.network/app-development/threshold-access-control-tac/integration-guide
+      variant: EXTERNAL_SOLID
+    - label: Trust Overview
+      url: https://docs.threshold.network/app-development/threshold-access-control-tac/trust-assumptions/cbd-mainnet-version
+      variant: EXTERNAL_OUTLINE
+  rowReverse: false
+tacoFeaturesInfo:
+  title: "The Fully Decentralized End-to-End Encryption"
+  description: "Empower users to share data with qualifying recipients, and no-one else Encryption via a decentralized cryptosystem."
+  highlighted: ["Decentralized", "Encryption"]
+tacoFeatures:
+  - title: "Web3 needs TACo"
+    cards:
+      - title: "Decentralized Storage"
+        description: "TACo is the perfect compliment to persistent storage, finally enabling users to grant future access without having to download & re-encrypt locally."
+        icon: /images/storage-icon.svg
+      - title: "Web3 Social Networks"
+        description: "TACo enables many-to-many data sharing, minus a centralized  authority that eventually decides to harvest and monetize interpersonal communication."
+        icon: /images/social-icon.svg
+      - title: "Media content-gating"
+        description: "Whether it's a stream, a movie or a collectible, TACo enforces decryption rights to media, so owning an NFT has meaning beyond a symbolic receipt."
+        icon: /images/media-icon.svg
+  - title: "The Web needs TACo"
+    cards:
+      - title: "Hyper-sensitive data"
+        description: "From reproductive health data to  seed phrase recovery, TACo is leveraged to ensure that high-stakes data sharing doesn’t involve trusting an intermediary."
+        icon: /images/locker-icon.svg
+      - title: "IoT & Connected Vehicles"
+        description: "TACo is the antidote to the interconnected hardware panopticon, where every waking (and sleeping) moment of our lives is surveilled."
+        icon: /images/iot-icon.svg
+      - title: "Activism & Human Rights"
+        description: "TACo protects those most at risk of intimidation, censorship and reprisal based on a data leaks – whether via a hack, subpoena, or a data vending business model."
+        icon: /images/activism-icon.svg
+tacoDiagramsInfo:
+  title: "Integrate a Privacy Layer with Nothing to Hide"
+  description: "TACo isn't just auditable, it's comprehensible, from high-level schematic to client code."
```

Comment:
![image](https://github.com/threshold-network/website/assets/2332199/fd434fe1-9af8-4463-ac16-31de40af4d26)
 
This subtitle is formatting a bit weirdly. Maybe this small edit will help?

```suggestion
  description: "TACo isn't just auditable, it's comprehensible. From high-level schematic to client code."
```